include/NovelRT/Graphics/TextRect.h

```diff
@@ -1,50 +1,51 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
 
-#ifndef NOVELRT_NOVELTEXTRECT_H
-#define NOVELRT_NOVELTEXTRECT_H
-#include "NovelRenderObject.h"
-#include "RGBAConfig.h"
-#include <string>
-#include "GraphicsCharacterRenderData.h"
-#include "NovelImageRect.h"
-#include "NovelLoggingService.h"
-#include "ShaderProgram.h"
-
-namespace NovelRT {
-  class NovelTextRect : public NovelRenderObject {
+#ifndef NOVELRT_GRAPHICS_TEXTRECT_H
+#define NOVELRT_GRAPHICS_TEXTRECT_H
+#ifndef NOVELRT_H
+#error Please do not include this directly. Use the centralised header (NovelRT.h) instead!
+#endif
+
+namespace NovelRT::Graphics {
+  class TextRect : public RenderObject {
   private:
     void reloadText();
 
     std::string _fontFileDir;
     std::string _previousFontFileDir = "";
     std::string _text = "";
-    std::vector<NovelImageRect*> _letterRects;
+    std::vector<ImageRect*> _letterRects;
```

Comment:
this should probably be a vector of unique_ptr (although eventually we want a single imagerect anyways)